<ui version="4.0" >
 <class>lp_ladspa_manager_item</class>
 <widget class="QWidget" name="lp_ladspa_manager_item" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QPushButton" name="pb_remove" >
     <property name="text" >
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="pb_select" >
     <property name="text" >
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QPushButton" name="pb_activate" >
     <property name="text" >
      <string>Activate</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="pb_edit" >
     <property name="text" >
      <string>Edit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
